sql/update_bene_monthly_proc.sql

```diff
@@ -0,0 +1,41 @@
+CREATE OR REPLACE PROCEDURE update_bene_monthly(
+	bene_id 					"BeneficiaryMonthly"."parentBeneficiary"%TYPE,
+	yr_month					"BeneficiaryMonthly"."yearMonth"%TYPE,
+	fips_cnty_code				"BeneficiaryMonthly"."fipsStateCntyCode"%TYPE,
+	medi_status_code 			"BeneficiaryMonthly"."medicareStatusCode"%TYPE,
+	buy_in_ind 					"BeneficiaryMonthly"."entitlementBuyInInd"%TYPE,
+	hmo_ind 					"BeneficiaryMonthly"."hmoIndicatorInd"%TYPE,
+	partc_contract_number_id 	"BeneficiaryMonthly"."partCContractNumberId"%TYPE,
+	partc_pbp_number_id 		"BeneficiaryMonthly"."partCPbpNumberId"%TYPE,
+	partc_plan_type 			"BeneficiaryMonthly"."partCPlanTypeCode"%TYPE,
+	partd_contract_number_id 	"BeneficiaryMonthly"."partDContractNumberId"%TYPE,
+	partd_pbp_number_id 		"BeneficiaryMonthly"."partDPbpNumberId"%TYPE,
+	partd_segment_num 			"BeneficiaryMonthly"."partDSegmentNumberId"%TYPE,
+	partd_retiree_mnthly 		"BeneficiaryMonthly"."partDRetireeDrugSubsidyInd"%TYPE,
+	partd_low_inc_cost_share 	"BeneficiaryMonthly"."partDLowIncomeCostShareGroupCode"%TYPE,
+	dual_elig_code 				"BeneficiaryMonthly"."medicaidDualEligibilityCode"%TYPE
+)
+AS
+$$
+    BEGIN
+    	UPDATE "BeneficiaryMonthly"
```

Comment:
See prev commonet r.e. INSERT trigger; if there's the possiblity of ever having an INSERT then this can be an UPSERT and I'll add in the INSERT trigger.

Reply:
I'd say either add support for it now, or set things to blow up explicitly if the condition is encountered. (I _think_ it'd fail as-is, which is good, but it's been a long day.)